pkg/datastore/datastore.go

```diff
@@ -196,6 +196,23 @@ func (sf SubjectsFilter) AsSelector() SubjectsSelector {
 	}
 }
 
+// SchemaDefinition represents a namespace or caveat definition under a schema.
+type SchemaDefinition interface {
```

Comment:
might be confusing due to the use of `definition` keyword in the schema language

Reply:
yeah, but this is already the terminology I use in the schemadsl package; this is just repeated here.